src/components/admin/dataTable/DataTable.tsx

```diff
@@ -0,0 +1,367 @@
+/** @jsxImportSource @emotion/react */
+import React from 'react';
+import { Table, Pagination } from 'antd';
+import { css, cx } from '@emotion/css';
+import {
+  rowCellStyle,
+  emptyRowStyle,
+  customRowHoverStyle,
+  lastRowNoBorderStyle,
+  EmptyTextWrapper,
+  TableWrapper,
+  TableTitleWrapper,
+  PaginationCenterWrapper,
+} from './DataTable.style';
+
+interface DataTableProps {
+  data: any[];
```

Comment:
이거 DataType으로 바꾸는건 어떨까요?
```suggestion
  data: DataType[];
```